packages/core/src/layout/Sidebar/UserSettings.tsx

```diff
@@ -48,6 +56,11 @@ export function SidebarUserSettings() {
           apiRef={githubAuthApiRef}
           icon={Star}
         />
+        <SidebarItem
```

Comment:
this needs to merge with the OIDCProviderSettings component above right?

Can come in a different PR but just confirming my understanding.